graalpython/com.oracle.graal.python/src/com/oracle/graal/python/builtins/modules/BuiltinFunctions.java

```diff
@@ -283,6 +291,224 @@ public Object absObject(VirtualFrame frame, Object object,
                 throw raise(TypeError, ErrorMessages.BAD_OPERAND_FOR, "", "abs()", object);
             }
             return result;
+       }
+    }
+
+    /**
+     * Base class for AllNode and AnyNode, as their logic and behaviors are very similar.
+     */
+    public abstract static class AllOrAnyNode extends PythonUnaryBuiltinNode {
+        enum NodeType { ALL, ANY }
+
+        /**
+         * Specializes on primitive storage types based on SequenceStorage.
+         */
+        @GenerateNodeFactory
+        public abstract static class AllOrAnySequenceStorageNode extends PythonUnaryBuiltinNode {
```

Comment:
Fixed, it not having this parent type originally messed with the compilation for some reason, I believe... It may very well have been something else that was causing the issue, though, considering it works fine right now. I made it inherit from `PNodeWithContext`.